src/sections/Cloud-Native-Catalog/header.js

```diff
@@ -0,0 +1,310 @@
+          right: 0px;
+          top: 35px;
+        }
+      }
+    }
+
+    @media screen and (max-width: 992px) {
+      padding-bottom: 100px;
+      .row {
+        flex-direction: column;
+        min-height: 40rem;
+        justify-content: space-around;
+      }
+      .catalog {
+        .calalog__card--programs {
+          left: 100px;
+          top: -60px;
+        }
+        .calalog__card--policies {
+          bottom: -50px;
+          right: 80px;
+        }
+        .calalog__card--patterns {
+          bottom: -60px;
+          left: 50px;
+          padding-left: 50px;
+          padding-right: 50px;
+        }
+        .calalog__card--webAssembly {
+          right: 80px;
+          top: -80px;
+        }
+      }
+    }
+
+    @media screen and (max-width: 767px) {
+      padding: 50px 0;
+      .header__detail {
+        display: flex;
+        flex-direction: column;
+      }
+      .header__title {
+        padding-right: 0;
+        text-align: center;
+        margin-bottom: 10px;
+      }
+      .header__title_description {
+        padding-right: 0;
+        text-align: center;
+        margin-bottom: 20px;
+      }
+      .header__button_section {
+        display: flex;
+        justify-content: center;
+      }
+      .catalog {
+        margin: 50px 0 0 0;
+        .calalog__card--child {
+          .calalog__card__title {
+            font-size: 18px;
+            line-height: 21px;
+          }
+        }
+        .calalog__card--webAssembly {
+          right: 20px;
+        }
+      }
+    }
+
+    @media screen and (max-width: 570px) {
+      .catalog {
+        .calalog__card--programs {
+          left: 50px;
+        }
+        .calalog__card--policies {
+          right: 50px;
+        }
+        .calalog__card--patterns {
+          left: 10px;
+        }
+        .calalog__card--webAssembly {
+          right: 50px;
+        }
+      }
+    }
+    @media screen and (max-width: 500px) {
+      .catalog {
+        margin: 180px 0 130px 0;
+        .calalog__card--policies {
+          bottom: -140px;
+        }
+        .calalog__card--webAssembly {
+          top: -140px;
+        }
+      }
+    }
+  }
+`;
+
+const Header = () => {
+  return (
+    <CatalogHeaderWrapper>
+      <div className="cloud-native-catalog header">
+        <Container>
+          <Row className="row">
+            <Col className="col header__detail" lg={6} md={12}>
+              <h1 className="header__title">Cloud Native Catalog</h1>
+              <p className="header__title_description">
+                Discover top-quality cloud native services for your
+                infrastructure with the Cloud Native Catalog. Browse now and
+                upgrade your cloud native operations!
+              </p>
+              <di className="header__button_section">
+                <Button primary title="Create Catalog" url="https://github.com/meshery/meshery.io/blob/master/README.md#add-your-patterns-to-the-catalog"></Button>
```

Comment:
it can be a self-closing tag
```suggestion
                <Button primary title="Create Catalog" url="https://github.com/meshery/meshery.io/blob/master/README.md#add-your-patterns-to-the-catalog" />
```